### PR TITLE
Rx observable exercise and Solution

### PR DESCRIPTION
**Pull request by sonukapoor**
Connects to #120

### CODE REVIEW

File: exercises/rxjs-observable/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Create an RxJS `Observable` to display 2 different integers using `setInterval` and `next`.
+
+## Exercise 2
+
+Using the `map` operator create an RxJS `Observable` that will take an array of numbers and multiply each item by 2. 
+
+## Exercise 3
+
+Using the `filter` operaor create an RxJS `Observable` that will take an array of numbers from 1 to 10 and filters it to return any number greater than 5.
```
Comment (rpdasilva) [inline exercises/rxjs-observable/README.md]: typo

File: exercises/rxjs-observable/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# RxJS Exercises
+
+## Exercise 1
+
+Create an RxJS `Observable` to display 2 different integers using `setInterval` and `next`.
+
+## Exercise 2
+
+Using the `map` operator create an RxJS `Observable` that will take an array of numbers and multiply each item by 2.
```
Comment (rpdasilva) [inline exercises/rxjs-observable/README.md]: To more clearly indicate what needs to be done, maybe reword to something similar: Create an observable from an array of numbers and using `map`, multiply each item by 2

File: exercises/rxjs-observable/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# RxJS Exercises
+
+## Exercise 1
+
+Create an RxJS `Observable` to display 2 different integers using `setInterval` and `next`.
```
Comment (rpdasilva) [inline exercises/rxjs-observable/README.md]: Should this be `setTimeout`?

File: exercises/rxjs-observable/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+## Exercise 4
+
+Create an RxJS `Observable` that will output a number and then throw an error. 
+
+## Exercise 5
+
+Create an Observable with the string `hello` and another one with the word `world`. 
+Merge the second observable's and print out `hello world`.
+
+## Exercise 6
```
Comment (rpdasilva) [inline exercises/rxjs-observable/README.md]: I find this difficult to determine what's expected.

Perhaps start by saying create an `Observable` from an array of number... then list the operations that should take place i.e. filter the first 3, then double the values, then...

File: exercises/rxjs-observable/README.md (hunk 3 above)
Comment (sonukapoor) [inline exercises/rxjs-observable/README.md]: Yes. Fixed.

File: exercises/rxjs-observable/README.md (hunk 1 above)
Comment (sonukapoor) [inline exercises/rxjs-observable/README.md]: Fixed

File: exercises/rxjs-observable/README.md (hunk 2 above)
Comment (sonukapoor) [inline exercises/rxjs-observable/README.md]: Changed.

File: exercises/rxjs-observable/README.md (hunk 4 above)
Comment (sonukapoor) [inline exercises/rxjs-observable/README.md]: Changed

Comment (rpdasilva) [inline exercises/rxjs-observable/README.md]: Oh, I actually meant this comment for Exercise 6, but I like the changes you made to #5 as well

I find 6 unclear

**Comment by rpdasilva** [comment]
:shipit:
